src/Models/PackageChangeRequest.php

```
@@ -0,0 +1,173 @@
+            [
+                ':organisation_id' => $organisationId,
+                ':package_price_id' => $packagePriceId,
+                ':status' => self::STATUS_PENDING,
+            ]
+        );
+        $row = $stmt->fetch();
+
+        return $row !== false ? self::hydrate($row) : null;
+    }
+
+    public static function find(Database $db, int $id): ?array
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)